tests/soapServer/index.js

```diff
@@ -0,0 +1,62 @@
+const http = require('http');
+const fs = require('fs');
+
+const _ = require('lodash');
+const SOAP = require('soap');
+
+const soapService = require('./soapService');
+
+const wsdlXML = fs.readFileSync(__dirname + '/regonline.wsdl', 'utf8');
+
+class SOAPServer {
+
+	constructor (port = 8000) {
+
+		this.port = port;
+		this.soapService = soapService;
+		this.wsdlXML = wsdlXML;
+
+		const server = this.server = http.createServer((request, response) => {
+			response.end('404: Not Found: ' + request.url);
+		});
+
+		this.soapServer = SOAP.listen(server, '/default.asmx', this.soapService, this.wsdlXML);
+
+		//Use this for debugging if needed
+		// this.soapServer.on('request', (req, methodName) => {
```

Comment:
Is there no flag that a developer could use to add this debug?

Reply:
Well this file is only for tests - so if a dev is actively adding/updating tests, they can uncomment this